t/e2e/whisper/whisper_mailbox_test.go

```diff
@@ -482,19 +509,19 @@ func (s *WhisperMailboxSuite) requestHistoricMessages(rpcCli *rpc.Client, mailbo
 	reqMessagesResp := baseRPCResponse{}
 	err := json.Unmarshal([]byte(resp), &reqMessagesResp)
 	s.Require().NoError(err)
-	s.Require().Nil(reqMessagesResp.Err)
+	s.Require().Nil(reqMessagesResp.Error)
 }
 
 type getFilterMessagesResponse struct {
 	Result []map[string]interface{}
-	Err    interface{}
+	Error  interface{}
```

Comment:
Why the renaming from `Err` to `Error` here and below?

Reply:
The API was changed in WhisperV6: 'err' field in response was changed to 'error'. So I fixed our tests because at the moment they don't check any errors from API.

Reply:
Thx for info. Would like to know why they changed it.

Reply:
I think this is a standardization part the whisper protocol between Ethereum and Parity. And such json looks more clear and readable: '{"err": "some error msg"}' vs '{"error": "some error msg"}'

Reply:
Shure, for JSON based external protocols it looks better. But could have also been done with tags translating between `err` and `error`. But that's not our decision. 😅